cypress/elements/fileMenu/open.js

```diff
@@ -5,7 +5,7 @@ import { clickMenuBarFileButton } from '../menuBar'
 const openModalEl = '*[class^="MuiDialogContent"]' // TODO: Add data-test to open modal to target this better
 const openModalFooterEl = '*[class^="MuiTableFooter"]'
 const openModalToolbarEl = '*[class^="MuiToolbar-root"]'
-const createdByOthersEl = '[data-value="byothers"]'
+const createdByOthersEl = 'byothers'
```

Comment:
This won't work. `getBySel` only works with `data-test` props. So ideally this element should be given a `data-test` prop instead. But we're still using the old favourites dialog right, so if it's very tedious to add maybe leave it as it was before?

Reply:
Also, if everything still passes even though this test isn't working, we should add a check to make sure that the `searchAOByName` actually returned the right thing...

Reply:
Or did you actually add `data-test="byothers"` on the element? If so the prop should probably be renamed to better follow our naming convention.

Reply:
Reverted back. `data-value` is used in the component itself, so changing it will break the component.
The favourite dialog is going to be rewritten eventually, so I will leave it for now.